3 mukemmel_sayi.py

```diff
@@ -0,0 +1,13 @@
+def mukemmel(sayi):
+    adet = 0;
+    for i in range(1, sayi):
+        if sayi % i == 0:
+            adet += i
+    if adet == sayi:
+        print(sayi, " mukemmel sayidir.")
+    else:
+        print(sayi, " mukemmel sayi degil")
+        return
+for i in range (1000):
```

Comment:
Minik bir hata. `for i in range(1000)` ifadesi 0'dan başladığı ve 0'ı da saydığı için `0 mükemmel sayıdır` gibi bir çıktı alıyoruz fakat 0 mükemmel bir sayı değil. Mükemmel sayıları pozitif tam sayılar içinde arıyoruz bu yüzden `for i in range(1,1000)` ifadesi daha doğru olacaktır.